src/lib/y2partitioner/widgets/format_and_mount.rb

```diff
@@ -77,6 +72,13 @@ def contents
         )
       end
 
+    private
+
+      def dm_name_for(device)
```

Comment:
This is just a temporary copy&paste from `Planned::CanBeEncrypted` isn't it? The functionality should be moved to `Encryption` at some point. Right now is repeated in the proposal (`Planned` and the partitioner). Not asking you to move it now, just saying (a FIXME would not hurt).